config/settings/qa_aks.yml

```diff
@@ -29,7 +29,6 @@ authentication:
 basic_auth:
   enabled: true
 
-current_recruitment_cycle_year: 2025
 features:
   send_request_data_to_bigquery: true
   teacher_degree_apprenticeship: true
```

Comment:
As we discussed, if it's set to true in `Settings.yml` it will already be true in QA. 

If you want it to be something different in QA (i.e false) then you could override it here.